### PR TITLE
feat(copy): Update operator-day pages

### PR DESCRIPTION
**Pull request by Lukewh**
## Done

- Update copy as-per the copy doc

## QA

- Visit https://juju-is-508.demos.haus/operator-day
- ensure copy matches the [copy doc](https://docs.google.com/document/d/1ZFRGkZ4vvvMVOgq4MIsBuebHsA7Y0rTyiZ76qjQpOAg/edit#heading=h.ds2l5u33x0lt)
- Visit https://juju-is-508.demos.haus/operator-day/kubecon-eu-2023
- And ensure the top paragraph matches the copydoc

## Issue / Card

Fixes WD-8142

### CODE REVIEW

File: templates/operator-day/index.html [hunk 1]
```diff
@@ -25,41 +25,68 @@ <h2>What is Operator Day?</h2>
...
+      <p>It is the most significant event for cloud-native computing, receiving more than 15,000 registrations in the past. KubeCon is the flagship conference of the <a href="https://www.cncf.io/">Cloud Native Computing Foundation (CNCF)</a>.
         Canonical has inaugurated the Operator Day event at KubeCon conference in 2020. If you missed them, you could freely access recordings from the sessions <a href="https://www.youtube.com/@UbuntuOS/playlists">on YouTube</a>.</p>
 
-       <h3>Operator Day at KubeCon NA 2023</h3>
+       <h3>How was Operator Day at KubeCon NA 2023?</h3>
 
-       <p>Come and join our new edition of Operator Day colocated at KubeCon NA 2023 a virtual online event, free to attend</p>
+       <p>The most recent Operator Day covered the entire journey of an engineering team, exploring software operators with hands-on sessions. Based on feedback from previous attendees,  we covered the step-by-step creation of an operator for <a href="https://juju.is/">Juju</a>, called a charm, using the <a href="https://juju.is/docs/sdk">Charm SDK</a>. The programme included:</p>
```
Comment (anthonydillon) [inline templates/operator-day/index.html:36]: These looks should be relative.

**Review by anthonydillon — APPROVED** [review]
One small comment in the code to update them LGTM 👍

File: templates/operator-day/index.html (hunk 1 above)
Comment (Lukewh) [reply templates/operator-day/index.html]: Good catch, updated

**Review by steverydz — APPROVED** [review]
LGTM 👍

**Comment by mcjaeger** [comment]
I am sorry, I missed to remove a button - could that be removed please, it points to an outdated page.
Screenshot 2024-01-09 at 11 20 15

**Comment by mcjaeger** [comment]
+1 - LGTM

**Pull request merged**